feat: metamak browser detection & mobile responsiveness

amrtokmak opened the pull request.

amrtokmak committed:
feat: metamak browser detection & mobile responsiveness

cd4761 commented:
@amrtokmak Did you add mobile responsibility? It is not work well on my device(Window pc & Mac)

amrtokmak committed:
fix: mobile hidden and responsiveness

jason-h23 committed:
fix: use explicit RPC URLs to resolve CORS error

wagmi's default http() uses eth.merkle.io which doesn't support CORS.
Changed to use Tenderly RPC URLs (PUBLIC_MAINNET_RPC, PUBLIC_SEPOLIA_RPC).
